content/updates/2020-10/3-Better-manage-product-models-and-variants-in-the-grid.md

```diff
@@ -1,26 +1,49 @@
 ---
-pim_announcement_alt_img: ""
+pim_announcement_img: ./img/better-manage-product-models-and-variants-in-the-grid.png
+pim_announcement_alt_img: "New counters and display option in the product grid"
 pim_announcement_audience:
 - CE
 - EE
 ---
 
 # Better manage product models and variants in the grid
 
-TODO
+Do you work a lot with product models and variants? If yes, you will surely enjoy these simple but yet efficient improvements we made in the product grid: a new filter on the product type, a new display option...
+
 ## Filter on the product typology
-::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/.html"
+::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/enrich-products-variants.html#filter-on-product-typology"
 :::
 
+The new `Product typology` filter in the product grid allows you to choose to display only the simple products or only the variant products. It's pretty useful if you need to quickly find different variant products in different product models, for instance.
+
+![Product typology filter](../img/product-typology-filter.png)
+
 ## Product variant display option
```

Comment:
c'est pas "variant product display option" plutôt?

Reply:
Hm yes, you're right.
```suggestion
## Variant product display option
```